tenzir/tests/exec/functions/string/pad_left.txt

```diff
@@ -0,0 +1,33 @@
+{
+  basic: "     hello",
+  zeros: "00042",
+  no_pad: "hello world",
+  empty: "-----",
+  single: "..x",
+  unicode: "★★★★test",
+  zero_len: "",
+  neg_len: "",
+  multi_pad: null,
+  null_str: null,
+  null_len: null,
+}
+warning: `pad_left` expected single character for padding, but got `ab` with length 2
+  --> exec/functions/string/pad_left.tql:19:14
+   |
+19 |   multi_pad: "test".pad_left(10, "ab"),
+   |              ~~~~~~ 
+   |
+
+warning: `pad_left` expected (string, int), but got (null, int64)
+  --> exec/functions/string/pad_left.tql:21:13
+   |
+21 |   null_str: null.pad_left(5),
+   |             ~~~~ 
+   |
+
```

Comment:
Your function doesn't handle `null_type` correctly. `null` should have the same behavior as `null.string()`.

Reply:
Changed in 767ad4a29e7.